lib/matplotlib/tests/test_quiver.py

```diff
@@ -3,10 +3,14 @@
 
 import numpy as np
 import pytest
+import cartopy.crs as ccrs
```

Comment:
Tests are failing because we do not have cartopy in the test environment.  You need to simplify the example and not use cartopy.  Similarly for the new example.